src/main/scala/rps/RPSMoves.scala

```diff
@@ -0,0 +1,37 @@
+package rps
+
+
+/*
+
+Scala has a strange view of Enumerations, that is
+a little bit too much for the average use-case.
+
+The "agile-version" of enumerations are called
+"sealed trait extended by case objects".
+
+Here we define a RPSMoves ADT that will contain
+useful definitions and methods to craft RPS Moves.
+
+*/
+
+
+object RPSMoves {
+
+sealed abstract class RPSMove(
+    val icon : String,
+    val value : String) {
+
+    override def toString = icon
+  }
+
+    case object ROCK extends RPSMove("💎", "0")
```

Comment:
The convention is to use PascalCase for identifiers, so this would be `Rock`

Reply:
Gotcha! 🤜🏻